src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -377,13 +377,30 @@ def cleanRSE(self, rse):
                             self.logger.exception(msg, rse['name'], str(ex))
 
                     self.logger.info("RSE: %s, Dir: %s, filesDeletedSuccess: %s",
-                                      rse['name'], dirLfn, deletedSuccess)
-                    rse['counters']['filesDeletedSuccess'] += deletedSuccess
-
-                    # Now clean the whole branch
-                    self.logger.debug("Purging dirEntry: %s:\n", dirPfn)
-                    purgeSuccess = self._purgeTree(ctx, dirPfn)
-                    if purgeSuccess:
+                                      rse['name'], dirLfn, filesDeletedSuccess)
+                    rse['counters']['filesDeletedSuccess'] += filesDeletedSuccess
+
+                    # Now delete the whole branch
+                    # First try to delete the base directory:
+                    rmdirSuccess = False
+                    purgeSuccess = False
+                    try:
+                        self.logger.debug("Trying to remove Dir: %s", dirPfn)
```

Comment:
Given that it's at the directory granularity, it might be better to have it logged at the INFO log level. In addition, I would suggest rephrasing it to `Trying to remove directory:`

Reply:
done